stdlib/LinearAlgebra/src/tridiag.jl

```diff
@@ -272,6 +272,8 @@ julia> eigvecs(A, [1.])
 """
 eigvecs(A::SymTridiagonal{<:BlasFloat}, eigvals::Vector{<:Real}) = LAPACK.stein!(A.dv, A.ev, eigvals)
 
+svdvals(A::SymTridiagonal) = sort(abs.(eigvals(A)); rev=true)
```

Comment:
It's probably better with a destructive version
```suggestion
function svdvals!(A::SymTridiagonal)
    vals = eigvals!(A)
    return sort!(map!(abs, vals, vals); rev=true)
end
```
since that will automatically also give the non-destructive version.

Reply:
Good point.